study-builder/src/main/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
```

Comment:
Had to add this file to strip out the `SlackAppender` because StudyBuilder was complaining about it.